NetKAN.schema

```diff
@@ -84,11 +84,11 @@
             "type": "string",
             "oneOf": [
                 {
-                    "pattern": "^#/ckan/ksp-avc(/.*)?",
+                    "pattern": "^#\/ckan\/ksp-avc(\/.*)$",
```

Comment:
```suggestion
                    "pattern": "^#/ckan/ksp-avc(/.*)?$",
```

This capture group is optional, not required (which is why it's a group in the first place). This should be treated as valid:

```json
    "$vref": "#/ckan/ksp-avc",
```

Reply:
Same as above, `$` fixes what I wanted to fix with `?`, lgtm